qiskit/visualization/pulse/qcstyle.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# pylint: disable=missing-docstring
+
+
+class OPStyleSched:
```

Comment:
Normally I like to keep one logical change per PR, just because it makes it easier to review and revert/bisect if something goes wrong. But I think since we have to update all the imports anyway this is fine. I'll push an update for this shortly